### DISCUSSION TITLE
How long does Lighthouse to analyse a URL

### DISCUSSION THREAD

**Discussion by SusanaRR** [GoogleChrome/lighthouse#14658] [Q&A] [answered]
Hi,

I'm using a standard configuration and I'm extracting only 7 metrics. A URL that takes to load around 2 seconds takes 8 to be analysed by Lighthouse. Is this time expected? Is there any way to reduce this time?

Thanks

**Comment by connorjclark** [comment, marked as answer]
Yes, this is expected. You can use `--only-audits` (using the CLI) or select just the Performance category (using devtools panel) to cut down on the time.

**Reply by SusanaRR** [reply]
Thanks @connorjclark 

Yes I have only performance and just few of the metrics from it.
